README.md

```diff
@@ -302,6 +304,22 @@ ws.on('message', function incoming(data) {
 });
 ```
 
+### Use the Node.js streams API
+
+```js
+const WebSocket = require('ws');
+const createWebSocketStream = require('ws/stream');
```

Comment:
What was the reasoning behind putting it in a different file instead of as a separate export? e.g.

```js
const { createWebSocketStream } = require('ws')
```
```

Reply:
I did not want to pollute the `WebSocket` class with spurious attributes but we already do that for `Server`, `Receiver`, and `Sender` so I'm fine exposing it as a property of the `WebSocket` class if that is preferred.

Reply:
Yep, thats why I wondered if there was reason why this was exposed as separate file instead of added to the exports like the rest. For the sake of consistency it makes the most sense to add it as export.

Reply:
Ok, to be honest I would like to remove both `Receiver` and `Sender` from the `WebSocket` class and tell people to require `ws/lib/receiver` if they need to. I think Removing `Server` is too disruptive.

Reply:
I agree with that sentiment as I consider both of those classes internals. But that would however be a breaking change. But for things like `Server` I do think it makes sense to expose it as a named export instead of a separate file as it would be a public API endpoint.

Reply:
Agreed.